manifests/0000_12_etcd-operator_06_deployment.yaml

```diff
@@ -57,6 +57,8 @@ spec:
           value: quay.io/openshift/origin-etcd
         - name: OPERATOR_IMAGE
           value: quay.io/openshift/origin-cluster-etcd-operator
+        - name: CLI_IMAGE
+          value: quay.io/openshift/cli
```

Comment:
value: quay.io/openshift/**origin-**cli:latest

IIUC this needs to match exactly the value in manifests/image-references

Reply:
aligned image-references with the rest of the images.

Reply:
still missing the origin- prefix here